scripts/variantstore_wdl/ImportArrays.wdl

```diff
@@ -174,8 +174,9 @@ task LoadArrays {
       fi
       set +e
       bq show --project_id ~{project_id} $SAMPLE_LIST_TABLE > /dev/null
+      BQ_SHOW_RC=$?
       set -e
-      if [ $? -ne 0 ]; then
```

Comment:
I think previously this $? was checking the exit code of `set -e` (the previous command)... not sure how that ever worked?!  your variable is much more clear anyway